go.mod

```diff
@@ -4,96 +4,95 @@ go 1.20
 
 require (
 	github.com/MakeNowJust/heredoc v1.0.0
-	github.com/google/btree v1.0.1
+	github.com/google/btree v1.1.2
 	github.com/google/gofuzz v1.2.0
-	github.com/google/uuid v1.3.0
+	github.com/google/uuid v1.6.0
 	github.com/jteeuwen/go-bindata v3.0.8-0.20151023091102-a0ff2567cfb7+incompatible
-	github.com/openshift/api v0.0.0-20230620130318-b2e6ab3d6c69
-	github.com/openshift/apiserver-library-go v0.0.0-20230621110634-a99412818848
-	github.com/openshift/build-machinery-go v0.0.0-20230306181456-d321ffa04533
-	github.com/openshift/client-go v0.0.0-20230607134213-3cd0021bbee3
-	github.com/openshift/library-go v0.0.0-20230620084201-504ca4bd5a83
-	github.com/spf13/cobra v1.6.1
+	github.com/openshift/api v0.0.0-20231101013329-0d0d46454bb7
+	github.com/openshift/apiserver-library-go v0.0.0-20230503174907-d9b2bf6185e9
+	github.com/openshift/build-machinery-go v0.0.0-20220913142420-e25cf57ea46d
+	github.com/openshift/client-go v0.0.0-20230503144108-75015d2347cb
```

Comment:
these packages were downgraded for some reasons

Reply:
Yeah, I took the dependencies of your PR, without checking if they are older. It is easier to miss it, if one doesn't use semver.